src/exporters/clickhouse/request_builder_mapper.rs

```diff
@@ -13,8 +13,8 @@ use tokio::task::JoinHandle;
 use tower::BoxError;
 use tracing::error;
 
-// todo: This seems high?
-const MAX_CONCURRENT_ENCODERS: usize = 20;
+// todo: Make this configurable?
+const MAX_CONCURRENT_ENCODERS: usize = 8;
 
```

Comment:
I didn't see any significant change with this since most bottleneck occurs at the DB, so in theory this should lower CPU usage.